src/variables/_connection.scss

```diff
@@ -0,0 +1,9 @@
+// https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-data
+
+// prefers-reduced-data: reduce
+@custom-media --reduced-data (prefers-reduced-data: reduce);
+@custom-media --data-shortage (prefers-reduced-data: reduce);
```

Comment:
`data-shortage` is misleading. If there were a data shortage, nothing could be loaded anyway…

Remove the alias or find a better one.